cluster/validation.go

```diff
@@ -6,6 +6,8 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/rancher/rke/k8s"
+
```

Comment:
We should remove the space between imports on line 10

Reply:
fixed.